lib/importer.rb

```diff
@@ -38,21 +38,39 @@ def lookup_country(country_str)
   end
 
   def normalize_country(country_str)
```

Comment:
I guess we could examine the innards of IsoCountryCodes and copy the techniques they use, if they seem useful. We could also possibly remove the regexs and put the mapping in a config file.